src/app.ts

```diff
@@ -177,6 +194,7 @@ function view(state: IState, actions: any) {
         editorOpenChangeObservable,
         graphCommandObservable,
       ),
+      ContextMenu.view({menuOpen: true}, {}),
```

Comment:
use the showMenu state variable to conditionally show the context menu view (ternary operator with the Empty being the alternative thing to show on a false) -- see editor just below this line